include/hip/hcc_detail/hip_runtime_api.h

```diff
@@ -1363,6 +1371,61 @@ hipError_t hipMemcpyDtoHAsync(void* dst, hipDeviceptr_t src, size_t sizeBytes, h
 hipError_t hipMemcpyDtoDAsync(hipDeviceptr_t dst, hipDeviceptr_t src, size_t sizeBytes,
                               hipStream_t stream);
 
+__attribute__((visibility("hidden")))
+hipError_t hipModuleGetGlobal(void**, size_t*, hipModule_t, const char*);
+
+
+/**
+ *  @brief Copies the memory address of symbol @p symbolName to @p devPtr
+ *
+ * @param[in]  symbolName - Symbol on device
+ * @param[out] devPtr - Pointer to a pointer to the memory referred to by the symbol
+ * @return #hipSuccess, #hipErrorNotInitialized, #hipErrorNotFound
+ *
+ *  @see hipGetSymbolSize, hipMemcpyToSymbol, hipMemcpyFromSymbol, hipMemcpyToSymbolAsync,
+ * hipMemcpyFromSymbolAsync
+ */
+inline
+__attribute__((visibility("hidden")))
+hipError_t hipGetSymbolAddress(void** devPtr, const void* symbolName) {
+    //HIP_INIT_API(hipGetSymbolAddress, devPtr, symbolName);
```

Comment:
do we need `HIP_INIT_API` here? if not perhaps this line can be removed? same goes to the one in `hipGetSymbolSize`

Reply:
I have no idea, since I have no idea what those macros are supposed to do (beyond being very noisy), but probably? However, I'm not going to sort out how to deal with all that noise in this particular iteration (and someone more familiar with the thing should do it anyway). So this should stay there as a reminder it's disabled for now.

Reply:
HIP_INIT_API is called from all hip (public) APIs for initializing hip. This behavior is needed to match CUDA runtime APIs.

Reply:
`hipLaunchKernelGGL` not calling `HIP_INIT_API` is serious issue.  I think it might be reason why HIP is broken if the HCC runtime is set to lazy init.  Could we prioritize on fixing the `HIP_INIT_API` issue?